src/python/bot/fuzzers/engine_common.py

```diff
@@ -141,18 +145,21 @@ def generate_new_testcase_mutations_using_radamsa(
 
   radamsa_runner = new_process.ProcessRunner(radamsa_path)
   files_list = shell.get_files_list(corpus_directory)
-  if not files_list:
-    # No mutations to do on an empty corpus, bail out.
+  filtered_files_list = [
+      f for f in files_list
```

Comment:
nit: maybe remove `files_list` and line 147, use `shell.get_files_list(corpus_directory)` here directly

Reply:
Can skip, i like local vars :)